gloo/test/main.cc

```diff
@@ -12,7 +12,9 @@
 namespace {
 struct Initializer {
   Initializer() {
+#ifndef _WIN32
     signal(SIGPIPE, SIG_IGN);
+#endif
```

Comment:
Widen this `#ifndef` to cover the entire `Initializer` struct and use, there's no point in having it call the empty function on startup.